src/Writers/CSharpWriter/CSharpWriter.Code.cs

```diff
@@ -1029,7 +1029,7 @@ private void Write(EntityContainerConstructor entityContainerConstructor)
             _("partial void OnContextCreated();");
         }
 
-        public void Write(IEnumerable<OdcmField> fields)
```

Comment:
This should just be removed. The CSharp model included Odcm fields for completeness but did not take dependencies on the fields existing, IIRC.